content/docs/rfcs/57/README.md

```diff
@@ -0,0 +1,335 @@
+Content topics specified by application protocols can be distributed over these shards.
+The Status app protocols are assigned to shard cluster `16`,
+as defined in [52/WAKU2-RELAY-STATIC-SHARD-ALLOC](/spec/52/).
+
+[51/WAKU2-RELAY-SHARDING](/spec/51/) specifies three sharding methods.
+This document uses *static sharding*, which leaves the distribution of content topics to application protocols,
+but takes care of shard discovery.
+
+The 1024 shards within the main Status shard cluster are allocated as follows.
+
+## Shard Allocation
+
+| shard index   |    usage                         |
+|    ---        |    ---                           |
+|       0 - 15  |   reserved                       |
+|     16 - 127  |   specific (large) communities   |
+|    128 - 767  |   communities                    |
+|    768 - 895  |   1:1 chat                       |
+|   896 - 1023  |   media and control msgs         |
+
+Shard indices are mapped to pubsub topic names as follows (specified in [51/WAKU2-RELAY-SHARDING](/spec/51/)).
+
+`/waku/2/rs/<shard_cluster_index>/<shard_number>`
+
+an example for the shard with index `18` in the Status shard cluster:
+
+`/waku/2/rs/16/18`
+
+In other words, the mesh network with the pubsub topic name `/waku/2/rs/16/18` carries messages associated with shard `18` in the Status shard cluster.
+
+
+### Implementation Suggestion
+
+The Waku implementation should offer an interface that allows Status nodes to subscribe to Status specific content topics like
+
+```
+subscribe("/status/xyz", 16, 18)
+```
+
+The shard cluster index `16` can be kept in the Status app configuration,
+so that Status nodes can simply use
+
+```
+subscribe("/status/xyz", 18)
+```
+
+which means: connect to the `"status/xyz"` content topic on shard `18` within the Status shard cluster.
+
+## Status Communities
+
+In order to associate a community with a shard,
+the community description protobuf is extended as follows.
+
+```protobuf
+
+syntax = "proto3";
+
+message CommunityDescription {
+  // The Lamport timestamp of the message
+  uint64 clock = 1;
+  // A mapping of members in the community to their roles
+  map<string,CommunityMember> members = 2;
+  // The permissions of the Community
+  CommunityPermissions permissions = 3;
+  // The metadata of the Community
+  ChatIdentity identity = 5;
+  // A mapping of chats to their details
+  map<string,CommunityChat> chats = 6;
+  // A list of banned members
+  repeated string ban_list = 7;
+  // A mapping of categories to their details
+  map<string,CommunityCategory> categories = 8;
+  // The admin settings of the Community
+  CommunityAdminSettings admin_settings = 10;
+  // If the community is encrypted
+  bool encrypted = 13;
+  // The list of tags
+  repeated string tags = 14;
+  // index of the community's shard within the Status shard cluster
+  uint16 relay_shard_index = 15
+}
+```
```

Comment:
```diff

syntax = "proto3";

message CommunityDescription {
  // The Lamport timestamp of the message
  uint64 clock = 1;
  // A mapping of members in the community to their roles
  map<string,CommunityMember> members = 2;
  // The permissions of the Community
  CommunityPermissions permissions = 3;
  // The metadata of the Community
  ChatIdentity identity = 5;
  // A mapping of chats to their details
  map<string,CommunityChat> chats = 6;
  // A list of banned members
  repeated string ban_list = 7;
  // A mapping of categories to their details
  map<string,CommunityCategory> categories = 8;
  // The admin settings of the Community
  CommunityAdminSettings admin_settings = 10;
  // If the community is encrypted
  bool encrypted = 13;
  // The list of tags
  repeated string tags = 14;
  // index of the community's shard within the Status shard cluster
+  uint16 relay_shard_index = 15
}
```

Reply:
You could add an empty line before line 129 to separate the new field from other fields.